pkg/controller/plan/kubevirt.go

```diff
@@ -219,6 +219,37 @@ func (r *KubeVirt) GetImporterPod(pvc core.PersistentVolumeClaim) (pod *core.Pod
 	return
 }
 
+// Get the importer pods for a PersistentVolumeClaim.
+func (r *KubeVirt) GetImporterPods(pvc core.PersistentVolumeClaim) (pods []core.Pod, found bool, err error) {
+	pods = []core.Pod{}
+	if pvc.Annotations[AnnImporterPodName] == "" {
```

Comment:
how about making this more explicit with:
```suggestion
	if _, ok := pvc.Annotations[AnnImporterPodName]; !ok {
```